package-lock.json

```diff
@@ -1,8 +1,11622 @@
 {
     "name": "typescript-to-lua",
     "version": "0.36.0",
-    "lockfileVersion": 1,
+    "lockfileVersion": 2,
```

Comment:
FWIW that would mean that every contributor would have to use Node >=15, otherwise npm v6 would revert these lockfile changes. But not upgrading also isn't great, since then some contributors would have to rollback node. Alternatively we can wait for [corepack](https://github.com/nodejs/corepack) to land, and lock npm to v6 even on newer node versions.

Reply:
Yeah I wasn't sure what to do about this one. I have a new system and did a fresh install of node.

The changelog mentions that the new lockfile version is backwards compatible, but I have not been able to try it:

https://blog.npmjs.org/post/626173315965468672/npm-v7-series-beta-release-and-semver-major

>package-lock.json is updated to a newer format, using "lockfileVersion": 2. This format is backwards-compatible with npm CLI versions using "lockfileVersion": 1, but older npm clients will print a warning about the version mismatch.

Reply:
Running `npm install` on v6 downgrades lockfile back to `"lockfileVersion": 1` and removes new key, which I wouldn't really call backwards compatible. But I guess that's okay